day-03/part-1/troisdiz.go

```diff
@@ -0,0 +1,77 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+    "strings"
+	"time"
+)
+
+func parseData(input string) [][]bool {
+    var result [][]bool
+    for _, line := range strings.Split(input, "\n") {
+        symbols := make([]bool, len(line))
+        for idx, letter := range line {
+            symbols[idx] = (letter == '#')
+        }
+        result = append(result, symbols)
+    }
+    /*
+    for _, line := range result {
+        for _, cell := range line {
+            if cell {
+                fmt.Print("#")
+            } else {
+                fmt.Print(".")
+            }
+        }
+        fmt.Print("\n")
+    }
+    */
+    return result
+}
+
+func puzzle(input [][]bool, right int, down int) int {
+    height := len(input)
+    width := len(input[0])
+
+    fmt.Printf("Height : %d, width : %d\n", height, width)
```

Comment:
Leftover print

Reply:
Indeed, I saw it and added a fix for this 👍

Reply:
Oh it seems I forgot one :-(, fixing